guides/mounts.md

```diff
@@ -19,10 +19,6 @@ You have to restart Wings to apply new changes to your Wings config.
 
 You have to configure mounts in admin Panel in order to use them with your servers. They consist of a source pad on the node and a target path where it will be mounted in the container.
 
-:::warning Path in the container
-Mounts cannot be mounted to or inside of `/home/container` or any subdirectory of it, nor can you cross-mount servers such as Server A's directory into Server B.
```

Comment:
Good, thanks! 

Could you amend this comment to specify that it is possible to do this (instead of not), and that the folder you want to mount into needs to exist for the mount to work.

Could you also change it to a "info" instead of a "warning".

Cheers!

Reply:
Sure. Done.